edgelet/edgelet-http/src/util/hyperwrap.rs

```diff
@@ -1,27 +1,42 @@
 // Copyright (c) Microsoft. All rights reserved.
 
+use crate::client::ClientImpl;
+use crate::error::{Error, ErrorKind, InvalidUrlReason};
+use crate::PemCertificate;
 use failure::ResultExt;
 use futures::future;
 use hyper::client::HttpConnector;
 use hyper::{Body, Client as HyperClient, Error as HyperError, Request, Response, StatusCode, Uri};
 use hyper_proxy::{Intercept, Proxy, ProxyConnector};
 use hyper_tls::HttpsConnector;
+use native_tls::{Certificate as TlsCertificate, TlsConnector};
 use typed_headers::Credentials;
 use url::percent_encoding::percent_decode;
 use url::Url;
 
-use super::super::client::ClientImpl;
-use crate::error::{Error, ErrorKind, InvalidUrlReason};
+use openssl::x509::X509;
 
 const DNS_WORKER_THREADS: usize = 4;
 
 #[derive(Clone, Debug)]
 pub struct Config {
     proxy_uri: Option<Uri>,
     null: bool,
+    identity_certificate: Option<PemCertificate>,
+    trust_bundle: Option<PemCertificate>,
 }
 
 impl Config {
+    pub fn identity_certificate(&mut self, identity_cert: PemCertificate) -> &mut Config {
```

Comment:
Make this consistent with `fn trust_bundle`. Either both should take `Option` or both should take the actual value.

Reply:
Thanks made this more consistent.